chef-automate/README.md

```diff
@@ -1,260 +0,0 @@
-# Chef - Cloud Devops Solution Template
```

Comment:
I think that's just how it looks in the diff?  The files are deleted if you navigate to the branch.